.github/workflows/rules.yml

```diff
@@ -43,6 +42,7 @@ jobs:
         pip install .[qibojit,tests]
     - name: Test with pylint
       run: |
+        pip install pylint
```

Comment:
@scarrazza Could you explain why this edit solved the problem? I'm just curious, as I didn't manage to solve the problem by myself.

Reply:
There is version conflict between tensorflow and pylint for type_extensions. So this last install overrides the bad version.

Reply:
Ok, thanks. Then I guess that line 40 is also redundant, isn't it?

Reply:
Yes, thanks.